main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,269 @@
+  - Returns the `issuer` for the purse's `brand`. See the [Issuer](#issuers)
+  section for more details about what this means and how it is used.
+  - ```js
+	const purseIssuer = purse.getIssuer();
+	```
+- [`purse.getCurrentAmount()`](https://agoric.com/documentation/ertp/api/purse.html#purse-getcurrentamount)
+  - Returns the current `amount` object stored in the purse. Note that
+    purses can be empty.
+  - ```js
+  const { issuer } = produceIssuer('bucks');
+  const purse = issuer.makeEmptyPurse();
+  const currentBalance = purse.getCurrentAmount();
+  ```
+- [`purse.deposit(payment, optAmount)`](https://agoric.com/documentation/ertp/api/purse.html#purse-deposit-payment-optamount)
+  - Deposit the `amount` contents of a `payment` into this purse,
+    returning the `payment`'s `amount`. If the optional argument
+    `optAmount` does not equal the `payment`'s balance,  or if `payment`
+    is an unresolved
+    [promise](https://agoric.com/documentation/glossary/#promise),
+    it throws an error.
+  - ```js
+     const { issuer, mint, amountMath } = produceIssuer('bucks');
+     const purse = issuer.makeEmptyPurse();
+	 const payment = mint.mintPayment(amountMath.make(123));
+	 const bucks123 = amountMath.make(123);
+	 // Deposit a payment for 123 bucks into the purse. Ensure that this is the amount you expect.
+	 purse.deposit(payment, bucks123);
+	 const secondPayment = mint.mintPayment(amountMath.make(100));
+	 // Throws error
+	 purse.deposit(secondPayment, fungible123);
+     ```
+- [`purse.makeDepositFacet()`](https://agoric.com/documentation/ertp/api/purse.html#purse-makedepositfacet)
+  - Creates a deposit-only facet on the `purse` that can be given
+    to other parties. This lets them make a deposit, but not make
+    withdrawals, to your purse.
+  - ```js
+     const depositOnlyFacet = purse.makeDepositFacet();
+     // Give depositOnlyFacet to someone else. They can pass a payment
+     // that will be deposited:
+     depositOnlyFacet.receive(payment);
+     ```
+- [`purse.withdraw(amount)`](https://agoric.com/documentation/ertp/api/purse.html#purse-withdraw-amount)
+  - Withdraw the `amount` from this purse into a new
+    `payment`. Returns the new `payment`.
+
+In addition, other objects, such as `issuer`, have methods which take
+purses as arguments or return purses. In particular:
+- [`issuer.makeEmptyPurse()`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-makeemptypurse)
+  - Returns a new empty `purse` for storing an `amount` of the same `brand` as
+  the `issuer`.
+  - ```js
+	   const { issuer, mint } = produceIssuer('bucks');
+	   const purse = issuer.makeEmptyPurse();
+   ```
+
+## Payments
+
+![Payment methods](payment.svg)   
+
+Payments hold an `amount` of digital assets that are either about to
+be sent to another party or which is in transit to another
+party. Payments are linear, meaning that either a `payment` has its full
```

Comment:
Not sure what the distinction is between "about to be sent" or "in transit". In actuality, no digital assets are transferred until another party calls `claim(payment) or deposits the payment. 

Maybe let's say, payments hold an `amount` of digital assets which can be transferred to another party.